thunder/torch/__init__.py

```diff
@@ -1241,96 +1249,191 @@ def abs(a: NumberLike | TensorLike, /) -> Number | TensorLike:
     return clang.abs(a)
 
 
+@torchsymbol(torch.Tensor.abs_, is_method=True, tags=(prims.OpTags.IN_PLACE,))
+def abs_(a: NumberLike | TensorLike, /) -> Number | TensorLike:
+    return prims.copy_(abs(a), a)
```

Comment:
nit: maybe adding a decorator for an out-of-place op to register an in-place counterpart could be cleaner if realizable?